### ISSUE TITLE
[3.x] Transforms not working (Laravel Mix)

### ISSUE THREAD

**Issue by marcreichel** [tailwindlabs/tailwindcss#6432]
**What version of Tailwind CSS are you using?**

v3.0.1

**What build tool (or framework if it abstracts the build tool) are you using?**

- Laravel 8.75
- Laravel Mix 6.0.39

**What version of Node.js are you using?**

14.17.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Reproduction URL**

https://github.com/marcreichel/tailwind3-transforms-reproduction

Screenshot:

![image](https://user-images.githubusercontent.com/7645035/145727757-997149ae-cb09-49a6-b9e6-b6255133a10d.png)

**Describe your issue**

The "base transform" selector is not generated. Therefore any transform utilities do not work.
Basically the there should be at least the following css code:

```css
.rotate-90, .translate-x-1\/2 {
    /* --tw-... */
    transform: rotate(var(--tw-rotate)) translateX(var(--tw-translate-x)) /* and so on ... */;
}
```

But this is completely missing.

**Comment by adamwathan** [comment]
Just had two seconds to look but it appears this works if you switch to:

```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

...instead of doing the imports. Not sure why, will get it looked into this week though!

**Comment by marcreichel** [comment]
Confirmed! Thanks a lot!

Are the imports not the "go to" for SCSS anymore? Thought for SCSS we have to use the imports instead of the `@tailwind` directives.

But good to know it is working with the directives.

Again: Thanks a lot! 💙

**Comment by ivan-bragin** [comment]
Got the same issue. Tried to import tailwind base in every possible way(scss import, @tailwind base, import "tailwind/tailwind.css";). In my case @apply backdrop-blur doesn't work cause --tw-backdrop-filter is not defined. It seems that defaults are not being created. Project configuration is basically default freshly created vite + vue app with create-vue and added postcss

**Comment by little3201** [comment]
same too

**Comment by keithn** [comment]
in vite + vue + tailwind 3.0 I also see that it doesn't seem to generate the css classes, I can definitely see the classes update on the dom, it just doesn't show any generated css.  If I stop vite and restart it, then they get generated.

**Comment by ihappycoder** [comment]
Same issue with `gatsby`, in css I have `@apply md:translate-y-10;` and then in inspector I see
<img width="355" alt="Screenshot 2021-12-13 at 15 08 50" src="https://user-images.githubusercontent.com/11615346/145827214-0ae78d33-ccad-4d60-831c-67b9528a0d1b.png">

I have 

```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

in global.css

**Comment by adamwathan** [comment]
Most of these comments are not about the main issue here (which is specific to Sass imports), they are actually this issue:

https://github.com/tailwindlabs/tailwindcss/issues/6409#issuecomment-991812274


...which is that you cannot use `@apply` inside a CSS module/Svelte component/Vue component to apply a class that provides `@defaults` (internal detail).

**Comment by duyhunter1001** [comment]
I found issue if you don't open @import "tailwindcss/base", However you just need copy all variables css of tailwind and paste into a new file css, after this import file css in your application.
